### ISSUE TITLE
Upgrade from 5.0.7 -> 5.1.0 break et for me

### ISSUE THREAD

**Issue by aslpavel** [MisterTea/EternalTerminal#133]
When I was trying to connect to my host machine with updated eternal terminal, I got following error.

```
$ et $HOST
ERROR: unknown command line flag 'idpasskeyfile'
Error starting ET process through ssh, please make sure your ssh works first
```

I'm running ArchLinux with default `et.cfg`
```cfg
; et.cfg : Config file for Eternal Terminal
;

[Networking]
port = 2022

[Debug]
verbose = 0
silent = 0
logsize = 20971520
```

Package content:
```
$ pacman -Ql eternalterminal
eternalterminal /etc/
eternalterminal /etc/et.cfg
eternalterminal /usr/
eternalterminal /usr/bin/
eternalterminal /usr/bin/et
eternalterminal /usr/bin/etserver
eternalterminal /usr/bin/etterminal
eternalterminal /usr/bin/htm
eternalterminal /usr/bin/htmd
eternalterminal /usr/lib/
eternalterminal /usr/lib/systemd/
eternalterminal /usr/lib/systemd/system/
eternalterminal /usr/lib/systemd/system/et.service
```

Logs do not contain any errors. After looking through the code it looks like this piece is to blame, but I'm not sure [SshSetupHandler.cpp](https://github.com/MisterTea/EternalTerminal/blob/26bfaf259215612bfad22e7a1dd3aad4f77e416a/src/terminal/SshSetupHandler.cpp#L48-L49,L60-L64)
- `SshSetupHandler::SetupSsh` always passes `--idpasskeyfile` argument
- but `genCommand` can call either `etserver` or `etterminal`, if `etterminal` was not found (note I have `etterminal` on my system) and `etserver` does not have this option!

**Comment by ailzhang** [comment]
Hi @aslpavel  the error message indicates you have an old client. Could you double check if you have an old ET install in your PATH?

**Comment by jwoos** [comment]
I have the same issue. I'm running WSL client with Arch VM

client (WSL):
```
$ et --version
et version 5.1.0
```

server (arch VM):
```
$ et --version
et version 5.1.0

$ etserver --version
etserver version 5.1.0

$ etterminal --version
etterminal version 5.1.0
```

After going through the code I came to the conclusion that the issue resides in SshSetUpHandler

**Comment by tbodt** [comment]
The AUR package for eternalterminal installs etserver as etterminal, which is broken.

**Comment by MisterTea** [comment]
@tbodt Thanks!  That was the issue.  Fixed in arch package 5.1.0-2
